libs/api/src/main/java/de/cyface/api/ServerConfig.java

```diff
@@ -229,26 +225,7 @@ public String extractKey(final Vertx vertx, final Parameter keyParameter)
             return null;
         }
 
-        final var keyBuilder = new StringBuilder();
-        try (var keyFileInput = new BufferedReader(
-                new InputStreamReader(new FileInputStream(keyFilePath), StandardCharsets.UTF_8))) {
-            var line = keyFileInput.readLine();
-            while (line != null) {
-                line = keyFileInput.readLine();
-                if (line == null || line.startsWith("-----") || line.isEmpty()) {
-                    // noinspection UnnecessaryContinue
-                    continue;
-                } else {
-                    keyBuilder.append(line);
-                }
-            }
-        }
-
-        final var key = keyBuilder.toString();
-        Validate.notNull(key, String.format(
-                "Unable to load key for JWT authentication. Did you provide a valid PEM file using the parameter %s.",
-                keyParameter.key()));
-        return key;
+        return Files.readString(Path.of(keyFilePath));
```

Comment:
With RS256 the `----` lines are needed, or else I get the error "pem files do not contain enough lines".
The Vert.X dokumentation also injects the keys including these lines.